addons/sale_project/models/sale_order.py

```diff
@@ -46,12 +46,15 @@ def _compute_show_project_and_task_button(self):
         show_button_ids = self.env['sale.order.line']._read_group([
             ('order_id', 'in', self.ids),
             ('order_id.state', 'not in', ['draft', 'sent']),
-            ('product_id.detailed_type', '=', 'service'),
+            ('product_id.type', '=', 'service'),
         ], aggregates=['order_id:array_agg'])[0][0]
         for order in self:
             order.show_project_button = order.id in show_button_ids and order.project_count
             order.show_task_button = order.show_project_button or order.tasks_count
-            order.show_create_project_button = is_project_manager and order.id in show_button_ids and not order.project_count and 'service' in order.order_line.product_template_id.mapped('detailed_type')
+            order.show_create_project_button = is_project_manager\
+                and order.id in show_button_ids\
+                and not order.project_count\
+                and 'service' in order.order_line.mapped('product_type')
```

Comment:
```suggestion
            order.show_create_project_button = (
                is_project_manager
                and order.id in show_button_ids
                and not order.project_count
            )
```
AFAICS, the product type is already checked by `order.id in show_button_ids` :thinking: